lib/control/validation/token.js

```diff
@@ -58,14 +41,20 @@ exports.create = function(req, res, next, vault) {
     }
   });
 
-  send(tokenParams, req.document.imageId, vault,
-    function(err, status) {
-      if (err) { return next(err) }
-      if (!status.hasOwnProperty('auth')) {
-        return bad.request(res, 'Token Error');
-      }
-      req.auth = status.auth;
-      return next();
-    });
+  return vaultRequest(vault, '/v1/auth/token/create', 'POST', tokenParams, {imageId: req.document.imageId}).then((status) => {
+    if (!status.hasOwnProperty('auth')) {
+      throw Error('Token Error');
+    }
+    req.auth = status.auth;
+    return getTokenTiming(status.auth.accessor, vault);
+  }).then((data) => {
+    if (data.expiration_time <= data.creation_time) {
+      throw new Error('Token has already expired');
```

Comment:
Derp, I covered the error condition where `new Date` failed because of a missing value. I added a test case to cover 0 and negative values in 4f1ef9e.